platform-sdk/docs/core/dynamic-address-book/roster/glossary-of-terms.md

```diff
@@ -0,0 +1,148 @@
+# Roster Glossary of Terms
+![Event Window Terminology](./EventWindowTerminology.drawio.svg)
+![Roster Terminology Diagram](./RosterTerminologyDiagram.drawio.svg)
+## Round Terminology
+
+### Latest Consensus Round
+
+The most recent round to have reached consensus.  Equal to the pending consensus round - 1.
+
+### Pending Consensus Round
+
+The round that the consensus algorithm is currently attempting to reach consensus on. Equal to the latest 
+consensus round + 1.
+
+### Rounds Non-Ancient
+
+A setting. The number of rounds that have reached consensus that are not ancient.
+
+### Ancient
+
+An event is ancient if its `birthRound` is less than or equal to the`latest consensus round` - `rounds non-ancient`.
+
+### Minimum Round Non-Ancient
+
+`minRoundNonAncient` is the minimum round that is considered non-ancient.  This is equal to the `pending consensus 
+round` - `rounds non-ancient`.
+
+### Non-Ancient Event Window
+
+Specifies the window of events that are being considered in consensus.  The window specifies the following:
+* `latestConsensusRound` - the latest round that has reached consensus.
+* `minRoundNonAncient` - the minimum round that is considered non-ancient. 
+
+### Latest Handled Round
+
+The `latestHandledRound` is the latest round to have been completely handled and applied to the state by the node.  
+Note that the application of this round to state produces a new roster to become effective after a configured 
+`rosterOffset`.  see terminology below.  
+
+### Future Event Horizon (a.k.a. Max Gossip Round)  
+
+The `futureEventHorizon` is the maximum round for which the node has an effective roster.  It is also the 
+maximum round that this node can gossip about.  Events which have a birth round higher than the `futureEventHorizon` 
+will not be gossipped.    
+
+### Past Event Horizon (a.k.a. Min Gossip Round)
+
+The `pastEventHorizon`, equal to `minRoundNonAncient` is the minimum round for which the node has an effective roster. 
+It is also the minimum round that this node can gossip about.  Events which have a birth round lower than the
```

Comment:
> It is also the minimum round that this node can gossip about

Is this actually true? In the current system, nodes maintain the ability to send ancient-non-expired events to peers.

Reply:
good point, thank you.